swagger.yaml

```diff
@@ -1,75 +1,83 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
-schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
+  version: "0.0.1"
+  title: OSU Dining Balances API
 produces:
-  - text/plain
+  - application/json
 paths:
-  /:
+  /diningbalances/{osuID}:
     get:
-      summary: Get information
-      description: Build and runtime information about the application.
+      tags:
+        - balances
+      description: "Get a single event by ID"
+      produces:
+        - application/json
       parameters:
-        - $ref: '#/parameters/pretty'
+        - name: osuID
+          in: path
+          description: "ID requested user"
```

Comment:
Let's make this a bit more clear by specifying that we are referring to the 9-digit id.